manifests/upstream/canary-checker/helm/values.yaml

```diff
@@ -0,0 +1,14 @@
+image:
+  tag: 'latest'
```

Comment:
should thos be .canaryChecker.version ?

Reply:
It's patched to that via kustomize. I couldn't place it here because the helm chart validation refused to accept it